src/screens/create-wallet-mnemonic/create-wallet-mnemonic.tsx

```diff
@@ -86,9 +88,21 @@ export class CreateWalletMnemonicScreenComponent extends React.Component<
 
                     {[...Array(NR_MNEMONICS_SCREEN).keys()].map(i => (
                         <Text key={`mnemonic-${i + indexFrom + 1}`} style={styles.secretWord}>
-                            {`${i + indexFrom + 1}. ${this.state.mnemonic[indexFrom + i]}`}
+                            {`${i + indexFrom + 1}. ${
+                                this.state.unveilMnemonic
+                                    ? this.state.mnemonic[indexFrom + i]
+                                    : '**********'
+                            }`}
                         </Text>
                     ))}
+
+                    <Button
+                        style={styles.unveilButton}
+                        onPressIn={() => this.setState({ unveilMnemonic: true })}
+                        onPressOut={() => this.setState({ unveilMnemonic: false })}
```

Comment:
for `onPressOut` let's use a setTimeout to delay a little the hidden state. we can use 2-300 ms as delay. This way if the user taps briefly on unveil button it will be more clear what's happening.